sys/include/uart_stdio.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_uart_stdio UART stdio
+ * @ingroup     sys
+ *
+ * @brief       stdio init/read/write functions for UARTs
+ *
+ * @{
+ * @file
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+#ifndef UART_STDIO_H
+#define UART_STDIO_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief initialize the module
+ */
+void uart_stdio_init(void);
+
+/**
+ * @brief read len bytes from stdio uart into buffer
+ */
+int uart_stdio_read(char* buffer, int len);
```

Comment:
I don't think that doxygen will parse this correctly, leading to incomplete API documentation and warnings while building the documentation. Hence, the answer is no. Sorry, bro!